helm/values.yaml

```diff
@@ -24,8 +25,20 @@ container:
   allowPrivilegeEscalation: false
   restartPolicy: Always
   command: ["/bin/sh", "-c"]
-  args: ["scripts/wait-for/wait-for $(POSTGRES_HOST):5432 -- npm --no-update-notifier run migrate && node index"]
+  args: ["npm --no-update-notifier run migrate; node index"]
 replicaCount: 1
 minReadySeconds: 10
 maxSurge: 1
 maxUnavailable: 1
+readinessProbe:
+  path: /healthy
+  port: 3002
+  initialDelaySeconds: 30
+  periodSeconds: 20
+  failureThreshold: 3
+livenessProbe:
+  path: /healthz
+  port: 3002
+  initialDelaySeconds: 20
+  periodSeconds: 20
```

Comment:
The pod kept getting restarted due to the checks failing. Increasing these values stopped that happening. Investigations for the reasons why are still ongoing.

Reply:
That's fine. If the investigation looks likely to take a while, I would encourage raising a ticket on the backlog so it can be prioritised and tracked.

Reply:
It's likely because of the sequelize migrations.  Similarly the payment service takes over 30 seconds to start node, run the migrations if any pending and setup the message queues.